docs/_themes/clusterhq/layout.html

```diff
@@ -1,6 +1,4 @@
-{%- if meta is defined and meta.get('layout') == 'homepage' -%}
-{%- extends 'layout_homepage.html' -%}
```

Comment:
:+1: We discussed this and `layout_homepage.html` seems to be a legacy of previous documentation where the first documentation page was set up to look like part of the main website.